src/Stratis.Bitcoin/Controllers/NodeController.cs

```diff
@@ -391,8 +391,9 @@ public async Task<IActionResult> GetTxOutAsync([FromQuery] string trxid, uint vo
         [Route("stop")]
         public IActionResult Shutdown([FromBody] bool corsProtection = true)
         {
-            // Start the node shutdown process.
-            this.fullNode?.Dispose();
+            // Start the node shutdown process, by calling StopApplication, which will signal to
```

Comment:
Ah brilliant ofcourse this makes sense good spot @sondreb

Reply:
Thanks Dan! One of those issues that takes some time to understand, but ends up being the simplest changes in code ;-)

Reply:
Rpc method uses dispose as well. May need similar update.

Reply:
Fixed now in the RPC stop method, changes to same as API stop/shutdown.